index.html

```diff
@@ -1,13 +1,36 @@
 <!doctype html>
 <html lang="en">
-  <head>
+
+<head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite App</title>
-  </head>
-  <body>
+</head>
+
+<body>
     <div id="app"></div>
     <script type="module" src="/main.js"></script>
-  </body>
+
+    <div id="buttons" align="center">
+        <button>
+            <tooltip placement="topLeft" trigger="hover">HI :)</tooltip>
+            Put your mouse on me!
+        </button>
+        <button>
+            <tooltip placement="bottomRight" trigger="click">Click again!</tooltip>
```

Comment:
Second note.
You are using `placement` and `trigger` attribute, which is also undeclared as Web component. If you are using just simple element like `div`, you can set any attribute with `data-` prefix.
`<div data-placement="left">`

More than that, you can use special API to read these attributes like so:
`element.dataset.placement`

More info here:
https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes